src/node_file.cc

```diff
@@ -615,13 +615,13 @@ static void Link(const FunctionCallbackInfo<Value>& args) {
 
   int len = args.Length();
   if (len < 1)
-    return TYPE_ERROR("dest path required");
-  if (len < 2)
     return TYPE_ERROR("src path required");
+  if (len < 2)
```

Comment:
@thefourtheye These unrelated changes are from previous commit, I wonder why one commit by yorkie still remains in this PR